crates/router/src/services/api.rs

```diff
@@ -592,7 +592,6 @@ async fn handle_response(
 
             match status_code {
                 200..=202 | 302 | 204 => {
-                    logger::debug!(response=?response);
```

Comment:
Removed this redundant log line, since the response is being logged (with the same key) on line 589, before the `match` block.